allensdk/brain_observatory/ecephys/ecephys_session_api/ecephys_nwb_session_api.py

```diff
@@ -298,3 +313,17 @@ def _get_full_units_table(self) -> pd.DataFrame:
 
     def get_metadata(self):
         return self.nwbfile.lab_meta_data['metadata'].to_dict()
+
+
+def clobbering_merge(to_df, from_df, **kwargs):
+    overlapping = set(to_df.columns.values.tolist()) & set(from_df.columns.values.tolist())
```

Comment:
I think the `tolist()` part is superfluous? And `set(to_df.columns.values)` should be fine?

Reply:
```
>>> import pandas as pd
>>> df = pd.DataFrame({"a": [1, 2, 3], "b": [2, 3, 4]})
>>> set(df.columns)
{'a', 'b'}
```

:astonished:

Reply:
wonderful!